feature/experimental/p4rt/otg_tests/google_discovery_protocol_packetin_test/google_discovery_protocol_packetin_test.go

```diff
@@ -475,11 +502,15 @@ func (gdp *GDPPacketIO) GetTrafficFlow(ate *ondatra.ATEDevice, frameSize uint32,
 	ethHeader := flow.Packet().Add().Ethernet()
 	ethHeader.Src().SetValue(*gdp.SrcMAC)
 	ethHeader.Dst().SetValue(*gdp.DstMAC)
-	ethHeader.EtherType().SetValue(int32(*gdp.EthernetType))
+	ethHeader.EtherType().SetValue(int32(0x8100))
+
+	vlan := flow.Packet().Add().Vlan()
+	vlan.Id().SetValue(int32(vlanID))
+	vlan.Tpid().SetValue(int32(*gdp.EthernetType))
```

Comment:
I'm no OTG expert, so just a question about tpid and ethertype in this configuration context

In canonical terms, TPID is an indicator that dot1q frame follows and set as 0x8100; and then ethertype of the payload follows the vlan-id
I.e. pseudo-header looks like `<dst_mac> <src_mac> <tpid> <prio/cfi> <vlan_id> <ethertype> <payload>`

Are the tpid/ethertype values reversed here, or it is just an OTG syntax and 0x8100 will be first, and 0x6007 will be second in the actual frame?

Reply:
True. But when I actually print out the bytes of the packet I get:

```
<dst_mac> <src_mac> <0x8100> <prio/cfi> <vlan_id> <tpid> <payload>
```

Reply:
It may be a standard syntax because, I had to do the same thing in gopacket layers

Reply:
Okay, got it. Will assume it's just a syntax thing. Thx.